### DISCUSSION TITLE
SRT Hevc

### DISCUSSION THREAD

**Discussion by Pixel-47** [datarhei/restreamer#707] [Q&A]
Hello, the program is very interesting, I am currently using it.
I have a question because OBS Studio has released an update in which you can use the RTMP protocol with HEVC, are you working on implementing such an option in Restream that will make it work?

I have a second question, I managed to achieve a delay on SRT with HEVC with conversion via Restream up to 5 seconds and conversion with hardware support to H264, is it possible to achieve something and reduce the delay? Currently I tested a segment length of 1 second and a list size of 3 segments. This works with a good quality internet connection like fiber optic but causes problems when using a mobile internet connection. RTMP works better here, but the correct settings are the default ones.
I am looking for a solution to save data transfer while maintaining image quality at low bitrate and then transferring it to twitch or possibly to another Obs Studio in order to convert this image to the correct one for Twitch plus adding any additional options in this setting such as overlays, etc. Do you know any interesting solution?

Is it possible to change anything more than the interface, such as: creating a command by running Docker in Ubuntu Server?
I use this command because I use a laptop with an Intel graphics card (CPU i5 5300U): Start with hardware support for Intel VAAPI

Copy
docker run --detach --rm --name core --privileged --volume /opt/core/config:/core/config --volume /opt/core/data:/core/data --publish 8080:8080 - -publish 8181:8181 --publish 1935:1935 --publish 1936:1936 --publish 6000:6000/udp datarhei/restreamer:vaapi-latest

**Comment by ioppermann** [comment]
HEVC support with RTMP will come in the upcoming release (in 2-3 weeks).

When you encode to H264 and want smaller segments you have to adjust the encoded GOP length according to the target segment length. In order to achieve a lower latency, you can send the encoded stream the RTMP (in "Processing and Control") and when sending the stream to Twitch, use RTMP as source.
